x/token/keeper/msg_server_test.go

```diff
@@ -264,6 +265,106 @@ func (s *KeeperTestSuite) TestMsgIssue() {
 	}
 }
 
+func (s *KeeperTestSuite) TestMsgIssue2() {
```

Comment:
Why don't you integrate this tests with previous one?

Reply:
Oh, I did not remove previous one to reference when I develop. I'll remove it. Thank you.